mapswipe_workers/setup.cfg

```diff
@@ -9,7 +9,7 @@
 # These settings are compatible with Black
 ignore = W503, E203
 inline-quotes = double
-max-line-length = 88
+max-line-length = 119
```

Comment:
Any reason why you're changing this?

Reply:
Seems more natural to use around 120 due to our current screen sizes. :D